pytorch_vision_once_for_all.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: Once for All: Train One Network and Specialize it for Efficient Deployment
+summary: We propose to train a once-for-all (OFA) network that supports diverse architectural settings by decoupling training and search, thus achieving efficient inference across many devices and resource constraints, especially on edge devices.
+image: https://github.com/mit-han-lab/once-for-all/raw/master/figures/overview.png
+author: MIT Han Lab
+tags: [vision]
+github-link: https://github.com/mit-han-lab/once-for-all
+github-id: https://github.com/mit-han-lab
+featured_image_1: https://github.com/mit-han-lab/once-for-all/raw/master/figures/cnn_imagenet_new.png
+featured_image_2: no-image
+accelerator: cuda-optional
+---
```

Comment:
This section is not displayed as a table for a reason that I can't really understand. The results is that it does not appear on the pre-built page on netlify (see tests).
When opening the .md file on github, you should see the first section as a table.

Reply:
Fixed. This is because the missing quote on the title.